### PR TITLE
FLUID-6696: Ensure that ToC is shown when initialized.

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6696

### CODE REVIEW

**Comment by jobara** [comment]
@jhung could you please take a look at the changes here and confirm if the address the issue you identified?

File: src/framework/preferences/js/Enactors.js [hunk 1]
```diff
@@ -459,7 +459,11 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                 createOnEvent: "onCreateTOCReady",
                 options: {
                     listeners: {
-                        "afterRender.boilAfterTocRender": "{fluid.prefs.enactor.tableOfContents}.events.afterTocRender"
+                        "afterRender.boilAfterTocRender": "{fluid.prefs.enactor.tableOfContents}.events.afterTocRender",
+                        "onCreate.show": {
```
Comment (amb26) [inline src/framework/preferences/js/Enactors.js:463]: Either this is a bug in the ToC core implementation in which case this fix should go there, or else it is part of the enactor's workflow, in which case it should manifest as an extra call to {enactor}.applyToc - either way we should have a comment here explaining the situation and reference to FLUID-6696

File: tests/framework-tests/preferences/html/Enactors-test.html [hunk 2]
```diff
@@ -88,7 +88,7 @@ <h2 id="qunit-userAgent"></h2>
         </div>
 
         <div class="flc-tableOfContents">
-            <div class="flc-toc-tocContainer"></div>
+            <div class="flc-toc-tocContainer" style="display: none"></div>
```
Comment (amb26) [inline tests/framework-tests/preferences/html/Enactors-test.html:91]: Add a comment here explaining that this attribute is explicitly a test for FLUID-6696

Comment (jobara) [reply tests/framework-tests/preferences/html/Enactors-test.html:91]: I've added comments to both the test html and the enactor code.

**Pull request merged**